scvid/data/distributed_anndata.py

```diff
@@ -96,31 +96,29 @@ class DistributedAnnDataCollection(AnnCollection):
 
     def __init__(
         self,
-        filenames: Union[Sequence[str], str],
-        limits: Optional[Sequence[int]] = None,
-        shard_size: Optional[int] = None,
-        last_shard_size: Optional[int] = None,
-        max_cache_size: Optional[int] = None,
+        filenames: Sequence[str] | str,
+        limits: Iterable[int] | None = None,
+        shard_size: int | None = None,
+        last_shard_size: int | None = None,
+        max_cache_size: int = 1,
```

Comment:
It is best to avoid `None` value here since `LRU(max_size=None)` will set to its own default `max_size=128`.